compiler+runtime/include/cpp/jank/read/parse.hpp

```diff
@@ -12,33 +14,75 @@ namespace jank::runtime
 /* TODO: Rename file to processor. */
 namespace jank::read::parse
 {
-  static option<char> get_char_from_literal(native_persistent_string const &sv)
+  static option<native_persistent_string>
+  parse_character_in_base(native_persistent_string_view const &char_literal, int const base)
```

Comment:
I think that it's odd to keep the `\o` or `\u` prefix on this when parsing. We should strip it out using `substr` before calling.

Reply:
Let's be not use a string_view here, though. I'm thinking of removing them from the supported jank types entirely, since they're altogether too difficult to use. For example, if we kept the string_view as the arg and we changed our call sites to use `substr`, we'd have undefined behavior since `substr` returns a temporary string, which we then borrow with the string_view and that doesn't keep it alive. Much better to just use a ref to const persistent string.